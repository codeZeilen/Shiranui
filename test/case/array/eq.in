let a = [1,2,3];
let b = [1,2,3];
if a = b {
    print("[1,2,3] ok");
}

let c = [1,2,4];
if a /= c {
    print("[1,2,4] ok");
}
if not a = c {
    print("not [1,2,4] ok");
}
