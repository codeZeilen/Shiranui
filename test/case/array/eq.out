[1,2,3] ok
[1,2,4] ok
not [1,2,4] ok
